### DISCUSSION TITLE
Kimai 2 doesn't load plugin

### DISCUSSION THREAD

**Discussion by kiseiju04** [kimai/kimai#3535] [General]
I'm trying to develop a Kimai plugin, but I can't get Kimai to find my plugin.

the source code:
https://github.com/kiseiju04/kimai-plugin

**Comment by kevinpapst** [comment]
Did you read the docs to start with?
What did you try to make it work? 
How does your setup and file structure look like on your dev machine?
Do you think it is a wise idea to use the namespace `KimaiPluginBundle`?
